owncloudDomain/src/main/java/com/owncloud/android/domain/files/FileRepository.kt

```diff
@@ -45,7 +48,9 @@ interface FileRepository {
     fun getFilesWithSyncInfoAvailableOfflineFromAccountAsFlow(owner: String): Flow<List<OCFileWithSyncInfo>>
     fun getFilesAvailableOfflineFromAccount(owner: String): List<OCFile>
     fun getFilesAvailableOfflineFromEveryAccount(): List<OCFile>
-    fun moveFile(listOfFilesToMove: List<OCFile>, targetFile: OCFile)
+
+   // Returns files in conflict.
```

Comment:
This is not aligned with the rest of lines 😬. I wouldn't write a dot (`.`) at the end of the line in this short comments
```suggestion
    // Returns files in conflict
```
Please write the other comment in the same way